test/features/site_channel_verification_test.rb

```diff
@@ -31,6 +31,32 @@ class SiteChannelVerificationTest < Capybara::Rails::TestCase
     refute_content "Cancel"
   end
 
+  test "When bad ssl happens" do
+    Rails.application.secrets[:host_inspector_offline] = false
```

Comment:
Not sure if it's okay or not to make an actual HTTP request, but since this is technically "integration" I viewed it as fine. 

Thoughts?

Reply:
Our test suite thus far does not make any external http requests, I'd prefer if we could stub these.  With the external services within Brave (e.g. eyeshade, promo) we've just been using the `stub_request` method, but have also set up vcr cassettes to stub sendgrid requests, and stubbed dns responses for site verification in test/stubs/dnsruby/.

Reply:
Mocked it out - https://github.com/brave-intl/publishers/pull/1345/commits/19075a783167f44b10d0b0d74477b2098a82f08d :)